src/Semantic/CLI.hs

```diff
@@ -158,7 +158,7 @@ graphCommand = command "graph" (info graphArgumentsParser (progDesc "Compute a g
         (x:_) -> pure $! Project (takeDirectory x) blobs (Language.languageForFilePath x) mempty
         _     -> pure $! Project "/" mempty Language.Unknown mempty
 
-    allLanguages = mconcat . intercalate ["|"] . fmap (pure . show) $ [Language.Go .. Language.TSX]
+    allLanguages = intercalate "|" . fmap show . delete Language.Unknown $ [minBound .. maxBound]
```

Comment:
I’d prefer to elide the `delete` call; change the list comprehension to`[Language.Go .. maxBound]` for an equivalent expression. Other than that, this is more efficient, so 👍 once that’s fixed.